packages/account/src/Sections/Profile/LanguageSettings/__tests__/language-settings.spec.tsx

```diff
@@ -8,16 +8,17 @@ import { mockStore, StoreProvider } from '@deriv/stores';
 jest.mock('@deriv/shared', () => ({
     ...jest.requireActual('@deriv/shared'),
     isMobile: jest.fn(() => false),
+    TranslationFlag: { lang_1: () => <div>Language 1 Flag</div>, lang_2: () => <div>Language 2 Flag</div> },
 }));
 
 jest.mock('@deriv/translations', () => ({
     ...jest.requireActual('@deriv/translations'),
     getAllowedLanguages: jest.fn(() => ({ lang_1: 'Test Lang 1', lang_2: 'Test Lang 2' })),
 }));
 
-jest.mock('@deriv/components', () => ({
-    ...jest.requireActual('@deriv/components'),
-    Icon: jest.fn(() => <div>Flag Icon</div>),
+jest.mock('react-i18next', () => ({
+    ...jest.requireActual('react-i18next'),
+    useTranslation: jest.fn(() => ({ i18n: { changeLanguage: jest.fn() } })),
```

Comment:
Why are we mocking this? Is some testcase using it?

Reply:
Let me remove this 🙏